Branch-SDK/Branch-SDK/BranchViewHandler.m

```diff
@@ -105,6 +107,28 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
     return !isRedirectHandled;
 }
 
+- (void)webViewDidFinishLoad:(UIWebView *)webView {
+    NSCachedURLResponse *resp = [[NSURLCache sharedURLCache] cachedResponseForRequest:webView.request];
+    NSInteger status = [(NSHTTPURLResponse*)resp.response statusCode];
+    NSLog(@"Status: %ld",(long)status);
+    if (status == 200) {
+        if (self.pendingBranchView != nil && self.pendingWebview != nil) {
+            UIViewController *holderView = [[UIViewController alloc] init];
+            [holderView.view insertSubview:self.pendingWebview atIndex:0];
+            UIViewController *presentingViewController = [[[[UIApplication sharedApplication] windows] firstObject] rootViewController];
+            [presentingViewController presentViewController:holderView animated:YES completion:nil];
+            
+            [self.pendingBranchView updateUsageCount];
+            
+            if (self.branchViewCallback) {
+                [self.branchViewCallback branchViewVisible:self.pendingBranchView.branchViewAction withID:self.pendingBranchView.branchViewID];
+            }
+        }
+    }
```

Comment:
In case there is an error while loading the url , we should ,be calling back  `branchViewError` with an error code. Lets add `branchViewError` to `BranchViewControllerDelegate` and then call it whenever there is an error with the url loading